scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -60,10 +68,21 @@ import scala.reflect.ClassTag
  */
 package object hdfs {
 
+  implicit class ScioHdfsUtil(val self: ScioUtil) {
```

Comment:
Where's `isHdfs` called?
No need to use implicit class if the method is only used within this file?